src/store/features/resources/resourcesSlice.ts

```diff
@@ -0,0 +1,42 @@
+import { type PayloadAction, createSlice } from "@reduxjs/toolkit";
+import storage from "redux-persist/lib/storage";
+import { PURGE } from "redux-persist";
+import { clientSignOut } from "@/store/features/auth/authSlice";
+
+export interface ResourceData {
+  id: number;
+  teamMemberId: number;
+  url: string;
+  title: string;
+  createdAt: Date;
+  updatedAt: Date;
+  addedBy: { member: [object] };
```

Comment:
Put the actual types here. There's a interface in ideation slice you can use, the voyage member. Since this one doesn't include an id you'll have to use a pick utility type to get the appropriate types from that interface for this one.

Reply:
Talking about the member: [object] thing btw, not sure why it did multi line selection